### ISSUE TITLE
Agenda: Community Council (CC), 17 Dec 2024

### ISSUE THREAD

**Issue by cekickafa** [grincc/agenda#151]
Solicit suggestions for agenda items for the Governance meeting to be held on Tuesday **Dec 17** @ _19:30 UTC_ in grincoin#general channel on [Keybase](https://keybase.io/team/grincoin#general). Please comment to provide topics or suggestions.
(Meeting can be followed on @keybase-ro channel - GRIN community [discord server](https://discord.gg/5p4vCQY9km) via GRIN discord bridge).

## To Do List.
-  Grin GUI, MWixnet node testing.

## Proposed Agenda
-  [Donation to Grincoin.org](https://forum.grin.mw/t/donation-to-grincoin-org/11415)

**Comment by Anynomouss** [comment]
Lest rephrase agenda topic number two to:
Some community members request more detailed instructions with test cases to make it easier to get started mwixnet in layman terms.

No demanding 😉
